src/Symfony/Bundle/FrameworkBundle/Console/Descriptor/JsonDescriptor.php

```diff
@@ -272,6 +272,8 @@ private function getContainerDefinitionData(Definition $definition, bool $omitTa
             }
         }
 
+        $data['usages'] = null !== $builder && null !== $id ? $this->getServiceEdges($builder, $id) : [];
```

Comment:
Should it be displayed if empty?

Reply:
Yes it should. These describing commands are APIs, which we generally like much more when their output is consistent :)

Reply:
got it